MarharytaKalynchuk/HW3/task3.py

```diff
@@ -0,0 +1,8 @@
+a = 9000000
```

Comment:
one way of implementation:

a, b = b, a

Reply:
Thank you for your comments! 
Yes, I remember now this solution from the lecture.

Reply:
No at all)!